modules/media/ajax/FileDownload.php

```diff
@@ -20,21 +20,9 @@
     exit;
 }
 
-// Make sure that the user isn't trying to break out of the $path by using a relative
-// filename. No need to check for '/' since all downloads are relative to $basePath.
-$file = $_GET['File'];
-if (strpos("..", $file) !== false) {
-    error_log("ERROR: Invalid filename");
-    header("HTTP/1.1 400 Bad Request");
-    exit(4);
-}
-
-$downloadNotifier = new NDB_Notifier(
```

Comment:
@jstirling91 these lines were deleted accidentally, they need to remain in 17.1, I will issue a new PR to re-add them